pkg/ccl/changefeedccl/kvevent/blocking_buffer.go

```diff
@@ -210,39 +237,54 @@ var bufferEntryPool = sync.Pool{
 	},
 }
 
-var _ quotapool.Request = (*bufferEntry)(nil)
+func newBufferEntry(e Event, alloc int64) *bufferEntry {
+	be := bufferEntryPool.Get().(*bufferEntry)
+	be.e = e
+	be.alloc = alloc
+	be.next = nil
+	return be
+}
+
+// memRequest is a quotapool memory request.
+type memRequest int64
```

Comment:
The downside of this change is now you're going to allocate one of these every time. The nice thing about the `bufferEntry` was that they were pooled.